state/runtime/evm/optimized_stack.go

```diff
@@ -11,57 +11,60 @@ import (
 // OptimizedStack uint256 integers for improved operations on values on the
 // stack and minimizes heap allocations.
 type OptimizedStack struct {
```

Comment:
We can maybe consider turning `OptimizedStack` into a type of `[]uint256.Int`, now that it has no fields except `data`:

```suggestion
type OptimizedStack []uint256.Int
```

Reply:
Updated with latest commit.